### ISSUE TITLE
How can I infer?

### ISSUE THREAD

**Issue by gabrielcustodio** [microsoft/unilm#217]
Do I need retrain to infer? Or can I use pre trained model and infer on Document Image Classification?
Should I use LayoutML to use the pre-trained model and infer?
Why didn't I find a script to infer, but one to retrain.

**Comment by wolfshow** [comment]
@gabrielcustodio Yes, you need to fine-tune the pre-trained LayoutLM using your document iamge classification dataset.

**Comment by gabrielcustodio** [comment]
> @gabrielcustodio Yes, you need to fine-tune the pre-trained LayoutLM using your document iamge classification dataset.

:( 
So, I cannot to infer with a new image without labels.txt using a pre-trained LayoutLM?
